app/assets/javascripts/application.js

```diff
@@ -11,6 +11,7 @@
 // about supported directives.
 //
 //= require jquery
+//= require bootstrap-sprockets
```

Comment:
I have no idea how the building RPM works, but I've tried to follow bootstrap instructions to make it work correctly with Ruby. (https://github.com/twbs/bootstrap-sass#a-ruby-on-rails)
Feel free to change it as needed